webapp/QAFs.md

```diff
@@ -0,0 +1,344 @@
+Quality Assured Features (QAFs)
+===============================
+
+Contents:
+- About this software
+- Compatibility
+- Features
+
+____________________________________________
+
+About this software
+-------------------
+
+- Name: Panoptes
+- Type: web application
+- Release: v2.0.beta4
```

Comment:
My original intention in tying the list of QAFs with a release stamp was for this to enforce relevance, i.e. to say that the following set of QA'd features strictly relate to version X.y.z
In that way, if the document never got update (e.g. per PR or release) then the document would still be self-consistent, as far as guarantees go.